polytiming.js

```diff
@@ -7,16 +7,18 @@
           return part.split('=');
         }).reduce(function(l, r) {
           if (r[0] === paramName) {
-            return l.concat(r[1].split(','));
+            return l.concat(r[1].split(/(?:,|%2C)/));
```

Comment:
Maybe instead complicating the splitter, we should pass `r[1]` through `decodeURIComponent`. WDYT?